pkg/rc/fields/fields.go

```diff
@@ -89,7 +97,7 @@ func (rc RC) ToRaw() (RawRC, error) {
 		Manifest:        string(manifest),
 		NodeSelector:    nodeSel,
 		PodLabels:       rc.PodLabels,
-		ReplicasDesired: rc.ReplicasDesired,
+		ReplicasDesired: &rc.ReplicasDesired,
```

Comment:
scared me for a moment that if you change an `RC`'s `ReplicasDesired` that suddenly a `RawRC`'s `ReplicasDesired` can change, but luckily `rc` is passed by value, so it is all good.